pkg/svcat/service-catalog/plan.go

```diff
@@ -34,21 +34,36 @@ const (
 
 // RetrievePlans lists all plans defined in the cluster.
 func (sdk *SDK) RetrievePlans(opts *FilterOptions) ([]v1beta1.ClusterServicePlan, error) {
-	plans, err := sdk.ServiceCatalog().ClusterServicePlans().List(v1.ListOptions{})
-	if err != nil {
-		return nil, fmt.Errorf("unable to list plans (%s)", err)
+	// Filter by Broker
+	if opts.Broker != "" {
```

Comment:
You got rid of the check that `opts` was not nil. The tests are passing in nil. So, the tests are going to panic. Either check for nil here or have the tests pass in a `&FilterOptions{}` instead of nil.